src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,42 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String color;
+    private String model;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. It is a good practice to make fields private and provide public getter and setter methods for accessing them.